paddle/fluid/operators/gather_scatter_kernel.h

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/fluid/framework/tensor.h"
+
+#pragma once
+
+namespace paddle {
+namespace operators {
+
+#define Instantiate_Template_Funtion(func)                              \
+  Instantiate_Template_Funtion_index_t(                                 \
+      func, int) Instantiate_Template_Funtion_index_t(func, float)      \
```

Comment:
Could you split the macro so that each line has one `Instantiate_Template_Funtion_index_t`?

In that case the code would be beautiful.

Reply:
Done.